network/lib_sddf_lwip/lib_sddf_lwip.mk

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2022, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+ifeq ($(strip $(SDDF_LWIP_NUM_BUFFS)),)
+$(error MAX_NUM_BUFFS must be specified)
+endif
+
+CFLAGS += -DSDDF_LWIP_NUM_BUFFS=$(SDDF_LWIP_NUM_BUFFS)
+
+LIB_SDDF_LWIP_DIR := ${SDDF}/network/lib_sddf_lwip
+LIB_SDDF_LWIP_FILES := $(addprefix ${LIB_SDDF_LWIP_DIR}/, lib_sddf_lwip.c)
+LIB_SDDF_LWIP_OBJS := $(LIB_SDDF_LWIP_FILES:.c=.o)
+
+${LIB_SDDF_LWIP_OBJS}: | ${LIB_SDDF_LWIP_DIR} ${CHECK_FLAGS_BOARD_MD5}
```

Comment:
you need to check the datestamp for `${CHECK_FLAGS_BOARD_MD5}` not just its existance.
Do:
```
${LIB_SDDF_LWIP_OBJS}: ${CHECK_FLAGS_BOARD_MD5} | ${LIB_SDDF_LWIP_DIR}
```